src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+public class Event  extends tasks{
+    private static final int CommandLength=6;
+    public Event (String task){
+        super(task.substring(CommandLength));
+        System.out.println("Got it. I have add this task: ");
+        displayTasks();
+        System.out.println("Now you have "+taskNum+" tasks in the list.");
+        System.out.println("");
+    }
+    public  void displayTasks(){
+        System.out.print("  [E]");
+        if(done==true)
```

Comment:
For if-else class of statements, perhaps try adding { } for your code that follows :)